media/public/include/IMediaPipeline.h

```diff
@@ -1172,6 +1172,28 @@ class IMediaPipeline
      * @retval true on success false otherwise
      */
     virtual bool getVolume(double &volume) = 0;
+
+    /**
+     * @brief Set mute status of pipeline.
+     *
+     * Muting does not change the underlying volume setting so when
+     * unmuted the user will hear audio at the same volume as previously
+     * set.
+     *
+     * @param[in] mute   Desired mute state, true=muted, false=not muted
+     *
+     * @retval true on success false otherwise
+     */
+    virtual bool setMute(bool mute) = 0;
+
+    /**
+     * @brief Get current mute status of the pipeline
+     *
+     * @param[out] mute   Current mute state
```

Comment:
Should be a semicolon after 'mute', same in getMute description too. Could you also do the same in setVolume & getVolume.

Only been pernickety because its the interface :)